index.js

```diff
@@ -107,10 +107,28 @@ class bluetoothLE extends EventEmitter {
     }
   }
 
-  disconnect() {
+  async disconnect() {
```

Comment:
Is there a corresponding change that calls `disconnect` using `await` instead?

Reply:
Yep: https://github.com/tidepool-org/uploader/pull/991/commits/bb849503c73df40c6de6bb975a2778e1c078f8d0#diff-0f5ac093d02c581930f6d7548633e569R192